misc/dockerfiles/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+echo_green() { echo -e "\e[42m$@\e[0m"; }
+echo_red()   { echo -e "\e[101m$@\e[0m"; }
+
+echo_exec() { echo_green "$@"; "$@"; }
+
+DIR="$(cd "$(dirname $0)"; pwd)"
+ROOT_DIR="$(dirname $(dirname $DIR))"
+
+# copy the files for the Dockerfile into the docker dir for the build
+for f in Pipfile Pipfile.lock; do
+  cp $ROOT_DIR/$f $DIR/$f
```

Comment:
The variable expansions [should be quoted](https://mywiki.wooledge.org/Quotes).

Reply:
Added quotes